test/qtismtest/runtime/tests/RouteTest.php

```diff
@@ -601,10 +601,10 @@ public function testBranchUnknownTarget(): void
 
     public function testBranchToAssessmentItemRefOutsideOfTestPart(): void
     {
-        $route = self::buildSimpleRoute(Route::class, 2, 1);
-        $this->expectException(OutOfBoundsException::class);
```

Comment:
These test case should not be changed.

Reply:
Now, instead of throwing an exception, we are ignoring branching rules and moving to the next item (like there are no branching rules)